notebooks/env.ipynb

```diff
@@ -0,0 +1,628 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    !pip install gymnasium
</pre><p>gymnasium should be part of the RLHive's dependencies already so it doesn't need to be installed</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/chandar-lab/RLHive/pull/305/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>